test/unit/math/mix/fun/svd_U_test.cpp

```diff
@@ -0,0 +1,30 @@
+#include <test/unit/math/test_ad.hpp>
+
+TEST(MathMixMatFun, svd_U) {
```

Comment:
These tests compile for me but they fail when they run. Must be a bug somewhere. Comment them out and introduce them one by one till you find it.